example/src/screens/BasicPictureInPicture.tsx

```diff
@@ -17,9 +18,10 @@ function prettyPrint(header: string, obj: any) {
 export default function BasicPictureInPicture() {
   useTVGestures();
 
-  const pictureInPictureConfig = {
+  const pictureInPictureConfig: PictureInPictureConfig = {
     // Enable picture in picture UI option on player controls.
     isEnabled: true,
+    shouldEnterOnBackground: true,
```

Comment:
Please add a code comment here to outline why this is here.

e.g. `// Enable entering picture in picture mode when transitioning the application to the background`